packages/mobile/src/account/Settings.tsx

```diff
@@ -94,15 +100,23 @@ const mapDispatchToProps = {
   devModeTriggerClicked,
   setRequirePinOnAppOpen,
   toggleFornoMode,
+  setSessionId,
 }
 
 interface State {
   fornoSwitchOffWarning: boolean
 }
 
 export class Account extends React.Component<Props, State> {
+  componentDidMount = () => {
+    const sessionId = ValoraAnalytics.getSessionId()
+    if (sessionId !== this.props.sessionId) {
```

Comment:
When would ValoraAnalytics.getSessionId() be out of sync with what's in redux?

Reply:
Should only happen at the very start of a session.

Reply:
Ok it's not clear to me why the check is needed here but not elsewhere

Reply:
SessionId is set in the `ValoraAnalytics` class, which isn't connected to the redux store. What I am doing here is checking if the sessionId has been defined by `ValoraAnalytics` but not yet put in the store.

We really only need to do this once per session and we are doing it on the `Account` screen because that's the only place sessionId is displayed (but only when in dev mode).